carrot-log4j-aws-sns/src/main/java/com/carrotgarden/log4j/aws/sns/Appender.java

```diff
@@ -216,14 +225,14 @@ protected boolean ensureCredentials() {
 
             if (url != null) {
                 try {
-                    amazonCredentials = new PropertiesCredentials(url.openStream());
+                    amazonCredentials = new StaticCredentialsProvider(new PropertiesCredentials(url.openStream()));
```

Comment:
I'm hinky about using anything from com.amazonaws.internal, they don't even show up in their JavaDoc. Could we use ClasspathPropertiesFileCredentialsProvider(url) instead?